README.md

```diff
@@ -1241,23 +1244,28 @@ and the Dfinity project. [\[18\]][18]
 The Lightning Network is a proposed message relay network operating at a layer
 above the Bitcoin blockchain, enabling many orders of magnitude improvement in
 transaction throughput by moving the majority of transactions outside of the
-consensus ledger into so-called "payment channels". This is made possible (with
-great difficulty) by the Bitcoin scripting language, which enables parties to
+consensus ledger into so-called "payment channels".
+<!---
+lightning network exists beyond bitcoin
+--->
+Which enables parties to
 enter into stateful contracts where the state can be updated by sharing digital
 signatures, and contracts can be closed by finally publishing evidence onto the
 blockchain, a mechanism first popularized by cross-chain atomic swaps.  By
 openning payment channels with many parties, participants in the Lightning
-Network can become focal points for routing the payments of others, leading to a
-fully connected payment channel network, at the cost of capital being tied up on
-payment channels.
-
-While the Lightning Network can also easily extend across multiple independent
-blockchains to allow for the transfer of _value_ via an exchange market, it
-cannot be used to assymetrically transfer _tokens_ from one blockchain to
-another.  The main benefit of the GnuClear network described here is to enable
-such direct token transfers.  That said, we expect payment channels and the
-Lightning Network to become widely adopted along with our token transfer
-mechanism, for cost-saving and privacy reasons.
+Network can become focal points for routing the payments, and contracts of others, leading to a
+fully connected channel network, at the cost of capital being tied up on
+payment channels. This cost can be completely removed by making it into
```

Comment:
I said some confusing things. sorry.

The interest cost is real. 
How much would you pay in bitcoin right now for 10 bitcoin right now?
How much would you pay in bitcoin right now for 10 bitcoin in a month?
The difference between those 2 numbers is the interest cost for a month.

The value that is lost by the interest rate is destroyed. It cannot be given to gnut holders as profit. Even if you want to give an extra reward to gnut holders, you still shouldn't make their bonds do 2 things simultaneous. The security deposit against double signing should not also be used to decide who has power in consensus.

I think it is more complex to give consensus power to $10 of bonds to secure $100 of money. That is as hard as balancing a mattress on a book. 
It is simpler to let the owner of the $100 of money have the consensus power over their own funds. Like balancing a mattress on a mattress. 

I have never suggested that validators should be able to spend or use bonded tokens. That is the exact opposite of what I am saying.
I am suggesting that we use the bonded coins only as a security deposit against double-signing, and to not re-use it for choosing who has power in consensus.

>If we can find some use to those bonded reserve-pool collateral tokens, we'll implement it.
That is exactly what I am telling you not to do. The bonds should be used for security deposit against double-signing, and nothing else. 

>But I'm not convinced that channels are the solution
I am not telling you to use channels. That is one way to solve this problem, but it is not the only way to solve this problem.
I am telling you that you can't re-use bonds for multiple things.
DPOS could work too. Or rewarding everyone with their computer turned on, and having every node participate. 

If you use the same bonds to both secure against double-signing and to decide who has power in consensus, it will do both things poorly.
The size of bonds will be too big to optimally secure against double-signing which will waste money, and it will be too small to decide who should have power in consensus which will cause retirement attacks from validators.